CONTRIBUTING.md

```diff
@@ -0,0 +1,95 @@
+# Contributing to dash
+
+## Getting Started
+
+Refer to the [readme](README.md) for installation and development instructions.
+
+## Coding Style
+
+Please lint any additions to react components with `pylint` and `flake8`.
+
+## Pull Request Guidelines
+
+Use the [GitHub flow][] when proposing contributions to this repository (i.e. create a feature branch and submit a PR against the master branch).
+
+## Running the Tests
+
+**Warning:** _Tests do not currently run on windows._
```

Comment:
could we create an issue about this and then link the issue inside this `CONTRIBUTING.md` file? that way, a windows contributor has a place to ask us about how to get past this